spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-polling-binder-sample/pom.xml

```diff
@@ -11,7 +11,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>spring-cloud-gcp-pubsub-polling-binder-sample</artifactId>
-    <name>Spring Cloud GCP Stream Polling Pub/Sub Code Sample</name>
+    <name>Spring Cloud GCP Code Sample - Pub/Sub Stream Polling</name>
```

Comment:
Since we are renaming things for consistency -- `Spring Cloud GCP Code Sample - Pub/Sub Stream Binder Polling`

Reply:
Done.